scripts/specs/core/CBaseEntity.lua

```diff
@@ -1818,6 +1818,23 @@ end
 function CBaseEntity:getJobPointLevel(jpType)
 end
 
+---@param jobId integer
```

Comment:
CI is complaining about this:

```suggestion
---@param jobID integer
```